mayastor/src/bin/cli/nexus_cli.rs

```diff
@@ -67,6 +67,12 @@ pub fn subcommands<'a, 'b>() -> App<'a, 'b> {
                 .required(true)
                 .index(2)
                 .help("uri of child to add"),
+        )
+        .arg(
+            Arg::with_name("norebuild")
```

Comment:
instead of calling this norebuild -- why not call it "rebuild"

Reply:
I chose this for consistency with the rpc call (where the naming also came up as a talking point). Below is a snippet from the proto file:

`message AddChildNexusRequest {
  string uuid = 1;    // uuid of the nexus
  string uri = 2;     // URI of the child device to be added
  bool norebuild = 3;   // auto start rebuilding
}`

IIRC 'norebuild' was chosen because the default rpc value for this is false and by default we wish to perform a rebuild.